snuba/settings.py

```diff
@@ -85,12 +85,12 @@
 
 TURBO_SAMPLE_RATE = 0.1
 
-PROJECT_STACKTRACE_BLACKLIST = set()
+PROJECT_STACKTRACE_BLACKLIST: Set[int] = set()
 
-TOPIC_PARTITION_COUNTS: MutableMapping[str, int] = {}  # (topic name, # of partitions)
+TOPIC_PARTITION_COUNTS: Mapping[str, int] = {}  # (topic name, # of partitions)
 
 
-def _load_settings(obj=locals()):
+def _load_settings(obj: Any = locals()) -> None:
```

Comment:
```suggestion
def _load_settings(obj: MutableMapping[str, Any] = locals()) -> None:
```

You can verify like this:

```
$ python -m mypy -c 'reveal_type(locals())'
<string>:1: note: Revealed type is 'builtins.dict[builtins.str, Any]'
```

(You will need to also add `MutableMapping` to the import at the top in addition to the suggested change.)